backend/app/models/launched_study.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class LaunchedStudy < ApplicationRecord
-  belongs_to :study
+  belongs_to :study, counter_cache: true
```

Comment:
I believe I was doing the counter_cache wrong on the has_many and it should be on the belongs_to side [according to the docs](https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache) (the values seem to be populating correctly now)